src/inpod/netns.rs

```diff
@@ -118,8 +118,7 @@ mod tests {
     #[test]
     fn test_run_works() {
         if !crate::test_helpers::can_run_privilged_test() {
-            eprintln!("This test requires root; skipping");
-            return;
+            panic!("This test requires root - rerun with TEST_MODE=root; skipping");
```

Comment:
Should these panic messages include the `and BULD_WITH_CONTAINER=1` part too?

Reply:
I thought about that but it's orthogonal IMO - you can run with `TEST_MODE` root if you are running the container as privileged, the test doesn't care.